src/main/java/org/jabref/gui/mergeentries/MergeWithFetchedEntryAction.java

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
No empty line at the beginning of a file (not sure why this file was touched by you)